core/templates/pages/about-foundation-page/about-foundation-page.component.html

```diff
@@ -255,13 +242,16 @@ <h1>{{ 'I18N_ABOUT_FOUNDATION_PAGE_SECTION_SIX' | translate }}</h1>
     padding: 5em 10em;
   }
 
-  .about-foundation-page button {
-    background-color: #00645c;
-    border-radius: 4px;
-    height: 40px;
+  .about-foundation-page .about-foundation-page-button-area {
     margin: 30px auto;
   }
 
+  :host {
```

Comment:
This is new to me ... just checking, what does it do and why do we need it?

Reply:
It's because we are changing the style inside our primary button component, and since angular styles are encapsulated when in an html page like this one, it doesn't reflect in our primary button component unless using the `:host` selector